src/Components/FormsCard/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../styles/colors';
+
+export default StyleSheet.create({
+  container: {
+    width: 360, // styling width according to percentages does so relative to view width
```

Comment:
Where does this number come from? On my screen, this makes the form card wider than the other components on the screen. (We do need to standardize our styling for components on this screen though).

Reply:
When I set the component's width to "100%", as done in the `CaseSummaryCard`'s styling, I expect it to fill the container that holds all cards on the `CaseScreen`. However, it fills the entire screen like this and I'm not sure why: 
<image height=500 width=230 src="https://github.com/calblueprint/impact-fund/assets/66931067/b56e155d-262a-4938-a652-8c3178c8da33">

Reply:
I was also trying to look into this and couldn't figure out why. Can only tell that this happens whenever we use `justifyContent: 'space-between'` because we're running into the same issue with the case status bar